src/query/parallel/px_query_execute/px_query_job.hpp

```diff
@@ -0,0 +1,156 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * px_query_job.hpp - parallel query job
+ */
+
+#ifndef _PX_QUERY_JOB_HPP_
+#define _PX_QUERY_JOB_HPP_
+
+#include "thread_entry.hpp"
+#include <mutex>
+#include <condition_variable>
+#include <sys/stat.h>
+#include <vector>
+#include "system.h"
```

Comment:
```suggestion
#include <mutex>
#include <condition_variable>
#include <vector>
#include "system.h"
```
불필요한 헤더를 제거했습니다. 확인 부탁드립니다.

Reply:
https://github.com/CUBRID/cubrid/pull/6442/commits/bfdbc45020e3e278e8e8a8f36f788a890594ff55 에서 적용하였습니다.